utils/mad-compat.rb

```diff
@@ -0,0 +1,54 @@
+class MadCompat < Formula
+  desc "MPEG audio decoder (custom build for Endless Sky)"
```

Comment:
Based on the Homebrew docs for [acceptable formulae](https://docs.brew.sh/Acceptable-Formulae) and [casks](https://docs.brew.sh/Acceptable-Casks) (which I am [quite familiar with](https://github.com/Homebrew/brew/commits/master/docs?author=EricFromCanada)), it's unlikely that either would be accepted into `homebrew/core` or `homebrew/cask`. I'm also not aware of any [other taps](https://docs.brew.sh/Interesting-Taps-and-Forks) specifically for maintaining distribution-ready versions of libraries. Creating a [custom tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) is possible, as is automated [bottle creation](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/), but for just one or two dependencies, having a local formula file should be adequate — especially for a library that hasn't had a new release since 2004.

Also, when it comes to cross-platform open-source games supporting macOS, this project doesn't have much company in its support for older macOS versions. A lot of API changes and/or compiler features that I don't understand rely on at least macOS versions 10.13 or 10.14, especially if Metal or Swift is involved, and most other games have simply dropped support for versions older than that (e.g. [openttd](https://formulae.brew.sh/formula/openttd), [openrct2](https://formulae.brew.sh/formula/openrct2), [openemu](https://formulae.brew.sh/cask/openemu)).